docs/03_goclient/16_tracing.md

```diff
@@ -0,0 +1,64 @@
+# Tracing and Context Propagation
+
+## Tracing
+
+The Go client provides distributed tracing support through [opentracing](https://opentracing.io/). Tracing can be
+configured by providing a `opentracing.Tracer` implementation in `Client` and `Worker` instantiation. Tracing allows
```

Comment:
For a lot of these types, I'd prefer to link to their documentation on godoc. For instance, opentracing.Tracer could link to https://godoc.org/github.com/opentracing/opentracing-go#Tracer